velox/functions/prestosql/aggregates/RegisterAggregateFunctions.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/prestosql/aggregates/RegisterAggregateFunctions.h"
+
+namespace facebook::velox::aggregate::prestosql {
+
+extern void registerApproxDistincts();
+extern void registerApproxMostFrequents();
+extern void registerApproxPercentiles();
+extern void registerArbitraryAggregates();
+extern void registerArrayAggregates();
+extern void registerAverageAggregates();
+extern void registerBitwiseAggregates();
+extern void registerBoolAggregates();
+extern void registerChecksumAggregates();
+extern void registerCountAggregates();
+extern void registerCountIfAggregates();
+extern void registerCovarianceAggregates();
+extern void registerHistogramAggregates();
+extern void registerMapAggAggregates();
+extern void registerMapUnionAggregates();
+extern void registerMaxSizeForStatsAggregates();
+extern void registerMinMaxAggregates();
+extern void registerMinMaxByAggregates();
+extern void registerSumAggregates();
+extern void registerVarianceAggregates();
+
+void registerAllAggregateFunctions() {
+  registerApproxDistincts();
+  registerApproxMostFrequents();
+  registerApproxPercentiles();
```

Comment:
These three function names do not end with "Aggregates". Shall we make them consistent with others?

Reply:
I rename these three function names. Thank you.